modular_skyrat/modules/customization/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -71,42 +71,51 @@
 			if(aux_zone)
 				aux.color = "#[draw_color]"
 
+	if (!owner || is_pseudopart || !ishuman(owner))
+		return
+
+	var/mob/living/carbon/human/H = owner
+	//set specific alpha before setting the markings alpha
+	if (H.dna.species.specific_alpha && H.dna.species.limbs_id == species_id)
+		for (var/ov in .)
+			var/image/overlay = ov
+			overlay.alpha = H.dna.species.specific_alpha
```

Comment:
If you're setting the bodypart alpha, you can just use it without checking species again.